zebrad/src/commands.rs

```diff
@@ -9,13 +9,18 @@
 //!
 //! See the `impl Configurable` below for how to specify the path to the
 //! application's configuration file.
+// TODO: update the list of commands above or find a way to keep it
+// automatically up to date.
```

Comment:
Since these docs were auto-generated by the Abscissa skeleton as instructions on how to use Abscissa, I think we could just remove the incorrect details rather than commit to maintaining them?